### ISSUE TITLE
Doesn't work with github electron

### ISSUE THREAD

**Issue by scott-cornwell** [fergiemcdowall/search-index#209]
Uncaught LevelUPError: Failed to require LevelDOWN (Module version mismatch. Expected 46, got 14.). Try `npm install leveldown` if it's missing

This looks like it might be helpful:
http://electron.atom.io/docs/v0.32.0/tutorial/using-native-node-modules/

Haven't got it working yet but will post more if I do.

**Comment by mewwts** [comment]
As someone states in this thread: https://github.com/Level/leveldown/issues/190, leveldown has been installed with the version of node that npm uses. But to run on Electron, it actually has to be built using Electron's version. So indeed, this http://electron.atom.io/docs/v0.32.0/tutorial/using-native-node-modules/#the-npm-way should be enough to fix your issue. Let me know if you can get through this, or please submit a sample Electron app with a package.json and main.js so that I can more easily reproduce the issue. Have a great day, and thanks for posting :+1:

**Comment by scott-cornwell** [comment]
Good news! So I was able to get it working using the 'easy way'. I don't know why I didn't just try that first. It took a minute to two to run, but I'd highly recommend it for anyone having this issue! 

I did attempt it using the npm way but it didn't seem to have any effect. I set my electron version and all that, it seemed to run install ok but no change in error message. 

Thanks for your response!
